core/sdk/src/api/device/device_peristence.rs

```diff
@@ -93,6 +94,6 @@ impl ExtnPayloadProvider for DevicePersistenceRequest {
     }
 
     fn contract() -> RippleContract {
-        RippleContract::DevicePersistence
+        RippleContract::Storage(StorageAdjective::Local)
```

Comment:
General question: Is there currently anything preventing a contract from being associated with multiple "adjectives" if needed in the future?

Reply:
Yes currently contracts dont have adjectives and they often have one to one mapping with a request and a processor so its not very scalable. Adjectives solves all those dependency issues